src/Symfony/Installer/NewCommand.php

```diff
@@ -252,14 +268,13 @@ private function download()
             $progressBar->setProgress($downloaded);
         };
 
-        $client = new Client();
-        $client->getEmitter()->attach(new Progress(null, $downloadCallback));
+        $this->client->getEmitter()->attach(new Progress(null, $downloadCallback));
```

Comment:
the fact that you change the state of the client each time this method is called makes it unreliable if you use the client as a global dependency of the class:

- it will impact any later downloads for instance. 
- it will cause issues if the method is called several times (even if this case is unlikely as the logic is in a CLI command)